src/blocks/store/s3.clj

```diff
@@ -78,6 +78,17 @@
     client))
 
 
+(def ^:private sse-algorithms
+  {:aes-256 ObjectMetadata/AES_256_SERVER_SIDE_ENCRYPTION})
+
+
+(defn select-sse-algorithm
+  [algorithm]
+  (if-let [algorithm' (algorithm sse-algorithms)]
```

Comment:
This is generally not a safe way to do lookups; if this function is called with `nil`, this will throw a `NullPointerException`. You should use `(get sse-algorithms algorithm)` instead.

Also, 🏌️ `(if-let [x expr] x (throw ...))` => `(or expr (throw ...))`